[Enhancement]- Adding Stock Availability (#10)
vtex-apps/product-availability: README.md

```json
  "dependencies": {

> "vtex.product-availability": "0.x"
> }

Reviewer: Just fix this small regression, please

```suggestion
    "vtex.product-availability": "0.x"
  }
```

| `threshold`     | `Number` | DefineMinimum quantity that makes low stock message appear (if message is set). Default: 0    |
| `lowStockMessage`        | `String` | String to be shown to user when stock is lower than threshold. Should have {quantity} inside the given string, to be replaced for the threshold property. Example: \"Only {quantity} left!\". Leave empty to not show. Default: ""              |
| `highStockMessage`  | `String` | String to be shown when stock is higher or equal than threshold. If left empty, won\'t show. Default: ""                                                              |

> | `showAvailability`  | `String` | Enables the possibility to show the available items instead of lowStockMessage or highStockMessage. Accepted value "stock". Default "".                                                           |

Reviewer: ```suggestion
| `showAvailability`  | `'disabled' | 'stock'` | Enables the possibility to show the available items instead of lowStockMessage or highStockMessage. Default `'disabled'`.                                                           |
```

> | `showAvailability`  | `String` | Enables the possibility to show the available items instead of lowStockMessage or highStockMessage. Default `'disabled'`. To enable it, string should be `'stock'`                                                     |

Reviewer: This has the same problem as the schema. You should type it like:
```suggestion
| `showAvailability`  | `'disabled' | 'stock'` | Enables the possibility to show the available items instead of lowStockMessage or highStockMessage. Default `'disabled'`. To enable it, string should be `'stock'`                                                     |
```
